Workbooks/Azure Resources/Azure Resource Health/Azure Resource Health.workbook

```diff
@@ -0,0 +1,247 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "value::all"
+        ],
+        "parameters": [
+          {
+            "id": "b09a0fdd-5b98-4304-bb0d-0d2e71b0df9c",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscriptions",
+            "type": 6,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::1"
+            ],
+            "typeSettings": {
+              "limitSelectTo": 100,
+              "additionalResourceOptions": [
+                "value::1",
+                "value::all"
+              ],
+              "includeAll": false
+            }
+          },
+          {
+            "id": "bdfc7b21-2dd7-4619-9f9e-7b81b698ad9c",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resources",

```

Comment:
this parameter is called "Resources" but it really is resource types?  and i'm not sure how it works given that this gets used in the alerts resource type selector that way, but then the "subscriptions" parameter is passed as the resource parameter, so it doesn't seem to ever work for me?

you might want to change this from an arg query that returns all the resource types to querying ARM to get the resource types that resource health supports?

(if you swap out the queryType:1 and query lines in this param with:
```
            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"/providers/Microsoft.ResourceHealth\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2014-04-01\"},{\"key\":\"$expand\",\"value\":\"metadata\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.metadata.supportedResourceTypes\",\"columns\":[]}}]}",
            "queryType": 12,
```
you'll get a resource type dropdown that only has the resource types that the health query provider supports

Reply:
Please advise, I'm not sure how to do this correctly, I got a json error.

Reply:
Ok I think It's done